### ISSUE TITLE
Aud not playing sounds

### ISSUE THREAD

**Issue by sdfgeoff** [UPBGE/blender#238]
Aud module is broken on linux in #69f2568
Attempting to play a sound causes the failure "File couldn't be read" even if the file exists (os.path.isfile returns is True).

Example zip:
[audbug.zip](https://github.com/UPBGE/blender/files/504834/audbug.zip)

Works in vanilla 2.78

**Comment by youle31** [comment]
For me it works on windows (with the last master and the release build I made). Maybe a build flag forgotten?

**Comment by sdfgeoff** [comment]
Ah, sorry, forgot to mention. I'm running it on LInux. I cloned it from git, hit build, and it's what I got from that.

**Comment by panzergame** [comment]
@sdfgeoff : You have to enable WITH_OPENAL.
I tested with your file and an UPBGE version compiled today without issue.

**Comment by sdfgeoff** [comment]
It seems you also have to enable WITH_CODEC_FFMPEG

Somewhere there should be a list of suggested build flags for UPBGE as the defaults are missing some features  So far as I can tell, the non-default flags needed for a functional game engine are:
 - WITH_CODEC_FFMPEG
 - WITH_OPENAL
 - WITH_SDL

Any I missed?

**Comment by youle31** [comment]
I don't know if it is the same issue but last time I tried to build on Linux with a cmake downloaded on cmake website and the compilation flags were incorrect. Then I installed cmake from package manager, and the compilation flags were correct. I don't know why. I never had this issue on windows.
I can't tell you what flags are needed because we haven't the same flags on windows and linux. But yeah, it should be a good idea to make an exhaustive list of all flags to enable according to OS.
